acados/sim/sim_gnsf.c

```diff
@@ -1659,7 +1659,7 @@ int sim_gnsf(void *config, sim_in *in, sim_out *out, void *args, void *mem_, voi
             blasfeo_dgemv_n(nyy, nff, 1.0, &YYf, 0, 0, &ff_traj[ss], 0, 1.0, &yyss, nyy * ss,
                             &yy_traj[ss], 0);
 
-            if ((opts->jac_reuse & (ss == 0) & (iter == 0)) | (!opts->jac_reuse))
+            if ((opts->jac_reuse && (ss == 0) && (iter == 0)) || (!opts->jac_reuse))
```

Comment:
Ask @giaf why he likes & better ;)

Reply:
I guess its not so high performing, but seems save..